Makefile

```diff
@@ -205,9 +205,16 @@ endurance-tests: export KOPIA_EXE ?= $(KOPIA_INTEGRATION_EXE)
 endurance-tests: build-integration-test-binary $(gotestsum)
 	 $(GO_TEST) $(TEST_FLAGS) -count=1 -parallel $(PARALLEL) -timeout 3600s github.com/kopia/kopia/tests/endurance_test
 
-robustness-tool-tests: $(gotestsum)
+robustness-tests: export KOPIA_EXE ?= $(KOPIA_INTEGRATION_EXE)
+robustness-tests: build-integration-test-binary $(gotestsum)
```

Comment:
@redgoat650 Just confirming, this target is not invoked by default on CI, correct?

Reply:
@julio-lopez That's correct. The tool tests are invoked under the `travis-release` target, but the `robustness-tests` target is only invoked manually